### PR TITLE
Create PreviewLinkModal

### PR DESCRIPTION
**Pull request by benghaziboy**
@kand @collin

### CODE REVIEW

File: app/views/modals/preview-link-modal.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+<div class="modal-header">
+  <button type="button" class="close" ng-click="$dismiss();" aria-hidden="true">&times;</button>
+  <h4 class="modal-title">Preview Link Generator</h4>
+</div>
+<div class="modal-body">
+  This link will require loggin in to the CMS to view. It will show all content within this<br>
```
Comment (kand) [inline app/views/modals/preview-link-modal.html]: Put this text inside a `<p>` tag. Also, shouldn't need `<br>` as the browser figures out where the line break should occur.

File: app/components/special-coverage/special-coverage-edit/special-coverage-edit-directive.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ angular.module('specialCoverage.edit.directive', [
 ])
```
Comment (kand) [inline app/components/special-coverage/special-coverage-edit/special-coverage-edit-directive.js]: Add `copyButton` as a module dependency to aid future refactoring.

I think you're able to use it at the moment as an unfortunate side-effect of poor module dependency organization.

File: app/styles/main.less [hunk 3]
```diff
@@ -369,6 +369,10 @@ nav.navbar {
   margin-top: 15px;
 }
 
+.top25 {
+  margin-top: 25px;
```
Comment (kand) [inline app/styles/main.less]: Isn't this a style added elsewhere for the pin to homepage button already?

Comment (benghaziboy) [inline app/styles/main.less]: that one was margin 15

**Comment by kand** [comment]
Otherwise, once the travis tests pass :+1:

**Pull request merged**